### PR TITLE
feat(healthz): add liveness

### PR DESCRIPTION
**Pull request by whalecold**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by whalecold** [comment]
https://github.com/caicloud/vaquita/pull/757

File: cmd/controller/app/release.go [hunk 1]
```diff
@@ -60,6 +63,18 @@ func Run(s *options.ReleaseServer) error {
 	if err != nil {
 		return err
 	}
+	mux := http.NewServeMux()
+	healthz.InstallPathHandler(mux, "/healthz")
```
Comment (hezhizhen) [inline cmd/controller/app/release.go:67]: 这里为什么不直接用 InstallHandler

File: cmd/controller/app/options/options.go [hunk 2]
```diff
@@ -47,4 +49,5 @@ func (s *ReleaseServer) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 	fs.Int32Var(&s.ConcurrentStatusSyncs, "concurrent-status-syncs", s.ConcurrentStatusSyncs, "The number of status controller worker that are allowed to sync concurrently")
 	fs.DurationVar(&s.ResyncPeriod, "resync-period", s.ResyncPeriod, "ResyncPeriod describes the period of informer resync")
 	fs.DurationVar(&s.ReleaseResyncPeriod, "handler-resync-period", s.ReleaseResyncPeriod, "ReleaseResyncPeriod is the resync period to invoke informer event handler")
+	fs.Int32VarP(&s.HTTPPort, "httpPort", "p", 8080, "Port for liveness check")
```
Comment (hezhizhen) [inline cmd/controller/app/options/options.go:52]: 这个 port 只用于 healthz 的话，叫 http port 是不是太 general 了？

Comment (hezhizhen) [reply cmd/controller/app/options/options.go:52]: 我看 Kubernetes 中叫 healthzPort

Comment (whalecold) [reply cmd/controller/app/options/options.go:52]: 我都统一改下

Comment (hezhizhen) [reply cmd/controller/app/options/options.go:52]: fs.Int32Var(&c.HealthzPort, "healthz-port", c.HealthzPort, "The port of the localhost healthz endpoint (set to 0 to disable)")

File: cmd/controller/app/release.go (hunk 1 above)
Comment (whalecold) [reply cmd/controller/app/release.go:67]: InstallHandler 看了下是个语法糖，当时抄这里的代码 https://github.com/caicloud/leader-election-example/blob/b6b54ea1f1e9e0225648135cf42cbbe17ccc3c33/main.go#L116

File: cmd/controller/app/options/options.go [hunk 3]
```diff
@@ -47,4 +49,5 @@ func (s *ReleaseServer) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 	fs.Int32Var(&s.ConcurrentStatusSyncs, "concurrent-status-syncs", s.ConcurrentStatusSyncs, "The number of status controller worker that are allowed to sync concurrently")
 	fs.DurationVar(&s.ResyncPeriod, "resync-period", s.ResyncPeriod, "ResyncPeriod describes the period of informer resync")
 	fs.DurationVar(&s.ReleaseResyncPeriod, "handler-resync-period", s.ReleaseResyncPeriod, "ReleaseResyncPeriod is the resync period to invoke informer event handler")
+	fs.Int32Var(&s.HealthzPort, "healthzPort", 8080, "Port for healthz check")
```
Comment (hezhizhen) [inline cmd/controller/app/options/options.go:52]: 格式和上面保持一致，用 healthz-port

Comment (hezhizhen) [reply cmd/controller/app/options/options.go:52]: health check，不是 healthz check

**Comment by hezhizhen** [comment]
LGTM
@qianlei90

**Pull request merged**